docs/AboutUs.md

```diff
@@ -57,3 +57,14 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 * Role: Developer
 * Responsibilities: UI
+
+### Loke Jin Xue Aaron
+
+<img src="images/aaron-ljx.png" width="200px">
+
+[[github](http://github.com/johndoe)]
+[[portfolio](team/johndoe.md)]
+
+* Role: Developer
+* Responsibilities: 
```

Comment:
Hey Aaron, just a slight issue, for the AboutUs, I think that you did not fill in your Responsibilities. Please fill it up to allow us to merge into the master branch.

Reply:
Also, please remember to add in your In charge of component: Logic as a new line!